BomberMan/Brain/Levels.swift

```diff
@@ -54,7 +54,26 @@ class Levels {
     }
 }
 
-
+class MultiplayerLevels {
+    
+    enum MultipLevels: String {
+        
+        case duel = "WWWWWWWWWWWWW B U B WWW0 WBWBW UWWW B U U WWW BWUUUWB WWUB  U  BUWW BWBWBWB WWW U B B WWWU WBWBW MWWW B U B WWWWWWWWWWWWW"
+        case battle = "WWWWWWWWWWWWWWWW0B B BUB B BMWW WBWBWBWBWBW WW B U B B B U WWBWBWBWBWBWBWBWW U BBBUBBB U WWBWBWBWBWBWBWBWW U BBBUBBB U WWBWBWBWBWBWBWBWW U BBBUBBB U WWBWBWBWUWBWBWBWW U B B B U B WW WBWBWBWBWBW WWMB B BUB B BMWWWWWWWWWWWWWWWW"
+        
+        case mincingmachine = "WWWWWWWWWWWWWWWWWWWWWWBMMMW0  BBB   WMMMMWWUMMMW  WBWBW  WMMMMWWMMMMB BUBBBUB WMMMMWWMMMMW  WUWUW  WMMMMWWWBUMB   BUB   BUBBBWWM    B WUWUW B    MWW  B B BUBBBUB B B  WW WUWUW W W W WUWUW WW  BUB  BUBUB  BUB  WWBWBWUW  B B  WUWUWBWW  BUB  BUBUB  BUB  WW WUWBW W W W WBWUW WW  B B BUBBBUB B B  WWM    B WUWUW B    MWWWBUBW   BUB   WUBBBWWUMMMB  WUWUW  UMMMMWWBMMMW BUBBBUB WMMMMWWWMMMW  WBWBW  WMMMMWWUMMMW   BBB   WMMMMWWWWWWWWWWWWWWWWWWWWWW"
+    }
+    func getlevel(name: String) -> String {
```

Comment:
I think,  will be better to return optional value

Reply:
I hear you)